creds/src/main.rs

```diff
@@ -152,28 +158,29 @@ fn show_proof_size(show_proof: &ShowProof<CrescentPairing>) -> usize {
 }
 
 pub fn run_show(
-    base_path: PathBuf
+    base_path: PathBuf,
+    presentation_message: Option<String>
 ) {
     let proof_timer = std::time::Instant::now();    
     let paths = CachePaths::new(base_path);
     let io_locations = IOLocations::new(&paths.io_locations);    
     let mut client_state: ClientState<CrescentPairing> = read_from_file(&paths.client_state).unwrap();
     let range_pk : RangeProofPK<CrescentPairing> = read_from_file(&paths.range_pk).unwrap();
-    
+    let pm = presentation_message.as_deref().map(|s| s.as_bytes());
```

Comment:
How does pm get defined when presentation_message is None?  (same question for line 193)

Reply:
pm stays None in that case, and that becomes an empty string in the crypto layer (and the transcript code hashes a [0,0,0,0] length array)